packages/components/autocomplete/src/Autocomplete.tsx

```diff
@@ -75,6 +75,11 @@ export interface AutocompleteProps<ItemType>
    * @default false
    */
   clearAfterSelect?: boolean;
+  /**
+   * If this is set to `true` the dropdown menu will stay open after selecting an item
```

Comment:
```suggestion
   * If this is set to `true` the dropdown menu will close after selecting an item
```

This is what I wanted to do. The question is if it makes more sense to talk about the case where the menu stays open or the one where it closes.

Reply:
So my thoughts around this where:
I wanted to keep the prop name close to the "clearAfterSelect" one as well as not introducing a breaking change.
Which is why i named it "closeAfterSelect" we could also put the whole variable vice versa to have false as the default and name it "keepOpenAfterSelect" and then the default is false (no breaking change) and setting it to true is keeping the dropdown open.
So might be easier to read and use?